source/material_model/rheology/composite_visco_plastic.cc

```diff
@@ -256,6 +323,11 @@ namespace aspect
                            Patterns::Bool (),
                            "Whether to include Peierls creep in the rheological formulation.");
 
+        prm.declare_entry ("Include Drucker Prager plasticity", "true",
```

Comment:
In order to avoid confusion with other parameters in the drucker_prager rheology module, I wonder if she specify that the prm entries include some reference to "composite rheology"?

```suggestion
        prm.declare_entry ("Include Drucker Prager plasticity in composite rheology", "true",
```

Reply:
Sure, this is a good idea for all the `Include mechanism` parameters.